appservice/src/connectToGitHub.ts

```diff
@@ -103,14 +103,13 @@ async function showGitHubAuthPrompt(node: AzureTreeItem, client: SiteClient, con
 
             context.properties.githubLearnMore = 'true';
 
-            // tslint:disable-next-line:no-unsafe-any
-            opn('https://aka.ms/B7g6sw');
+            await openUrl('https://aka.ms/B7g6sw');
         }
     }
 
     if (input === goToPortal) {
         context.properties.githubGoToPortal = 'true';
-        node.openInPortal(`${client.id}/vstscd`);
+        await node.openInPortal(`${client.id}/vstscd`);
```

Comment:
Wait, is this an async method?

Reply:
It is after my recent PR in UI :)

But it honestly doesn't matter if you await opening a url. Only lint really cares